### PR TITLE
Add SteamOS

### PR DESCRIPTION
**Pull request by CarlTheObeseCat**
Recently Created SteamOS theme just adding to site.

### CODE REVIEW

**Comment by CarlTheObeseCat** [comment]
Yes i know i fucked it 3 times i fixed it lol.

File: src/themes.json [hunk 1]
```diff
@@ -603,6 +603,18 @@
...
+		"name": "SteamOS",
+		"repo": "https://github.com/rose-pine/SteamOS",
+		"maintainers": [
+			{
+				"name": "CarlTheObeseCat",
+				"url": "https://github.com/CarlTheObeseCat"
+			}
+		],
+		"keywords": ["OS"],
+		"variants": false
```
Comment (mvllow) [inline src/themes.json:617]: This should be above sublime text as far as ordering :)

**Review by mvllow — CHANGES_REQUESTED** [review]

**Comment by mvllow** [comment]
Can we rename the repo to `rose-pine/steam-os` to match our other repos?

**Comment by CarlTheObeseCat** [comment]
> Can we rename the repo to `rose-pine/steam-os` to match our other repos?

yep gimme a second

File: src/themes.json (hunk 1 above)
Comment (CarlTheObeseCat) [reply src/themes.json]: Done

**Review by CarlTheObeseCat — COMMENTED** [review]
done

**Review by mvllow — APPROVED** [review]

**Comment by mvllow** [comment]
Thank you 😌

**Pull request merged**